tests/acceptance/features/apiSpacesShares/shareSubItemOfSpaceViaPublicLink.feature

```diff
@@ -53,6 +53,8 @@ Feature: Share a file or folder that is inside a space via public link
       | share_type        | public_link   |
       | displayname_owner | %displayname% |
       | name              | <name>        |
+      | uid_file_owner    | %space_id%    |
```

Comment:
```suggestion
      | uid_file_owner    | %space_id%    |
```
is file owner id same as space id ? As far as i remember it is different than that of a space id. If different then we can name it something like `%file_owner_id%`.

Reply:
I would be say that is different format. 

`getSpaceIdByName` - gives me `01e1375f-9f56-4e71-bb9f-2e750db67385$6cf15f71-7900-4f97-9e3a-69a467f56c41`
`uid_file_owner` - gives me spaceId but shortly `6cf15f71-7900-4f97-9e3a-69a467f56c41`
`space_id` - gives me spaceId  in format `<storageid>$<spaceid>!<opaqueid>` - `01e1375f-9f56-4e71-bb9f-2e750db67385$6cf15f71-7900-4f97-9e3a-69a467f56c41!6cf15f71-7900-4f97-9e3a-69a467f56c41`